### DISCUSSION TITLE
Unlimited Cloud Functions?

### DISCUSSION THREAD

**Discussion by SudgeFunday** [AntlerVC/firetable#1294] [Support / Q&A] [answered]
Hello, 

I'm new to Rowy but im wondering it says on the Rowy website,   that Rowy gives unlimited cloud functions, Im confused by this. I thought Rowy was connected to Firebase and in Firebase u have to pay to use cloud functions, so I'm confused about what causes the cloud functions to be unlimited.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @SudgeFunday, when Rowy mentions unlimited cloud functions, it means that you can create and deploy any number of cloud functions within the Rowy platform, without incurring any additional costs specifically tied to the usage of cloud functions. Rowy provides this feature without imposing specific limits or separate charges for cloud function usage within its ecosystem.

Moreover, Google Cloud Platform (GCP) offers a generous free tier that is usually sufficient for small-scale applications. 
For more details about Cloud Run pricing, including any potential costs beyond the free tier, I recommend referring to the official Google Cloud Platform pricing documentation, specifically the Cloud Run pricing page at: https://cloud.google.com/run/pricing.

If you have any further questions or need additional information, please feel free to ask!

**Reply by SudgeFunday** [reply]
so without incurring additional costs, would this also mean pub sub-functions, i could run theoretically every minute of every day and not incur additional costs?

**Reply by SudgeFunday** [reply]
i dont mean to use cloud functions like this as i will be using a pub-sub to check the schedule once a day, but it would help me really understand if the unlimited cloud functions apply to this theory, as this is pretty much the main reason I'm using rowy?
